specs/public_inputs.md

```diff
@@ -0,0 +1,263 @@
+# Public Inputs
+
+The following list contains all the data used by an Ethereum client to
+calculate a block and to define this calculated block.
+
+## Block
+
+- Hash: 256 bits
+- Block Fields used in RLP:
+    - ParentHash: 256 bits
+    - UncleHash: 256 bits
+    - Coinbase: 160 bits
+    - Root: 256 bits (State Trie Root)
+    - TxHash: 256 bits (Txs Trie Root)
+    - ReceiptHash: 256 bits (Receipts Trie Root)
+    - Bloom: 256 bytes
```

Comment:
also need new bloom circuit to verify `bloom` field  in the future?

Reply:
That's one possibility!  Add a circuit that verifies the correct construction of the bloom filter, so that light clients can use the bloom filter without requiring trust on a node.  This would be useful for a proof of validity used by light clients.

But before we have this circuit we could set the bloom filter to 0 (either 256 bytes at 0, or a null), patch geth to do that as well; and not provide the bloom filter as part of the proof, so that a client either trusts a synchronized geth node to query logs, or synchronizes the state itself and calculates the bloom filters internally.